src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  width: 799px;
+  min-height: 410px;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
+  padding: 10px;
+  box-sizing: border-box;
+  margin: auto;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $one-day-width;
+    height: 100px;
```

Comment:
use `@mixin` to set square's size.
Keep your `$one-day width` as a parameter of mixin.